chat-components/src/common/Constants.ts

```diff
@@ -152,8 +152,9 @@ export const AriaLabels = class {
     public static readonly TurnSoundOn = "Turn sound on";
     public static readonly Minimize = "Minimize";
     public static readonly EmailChatTranscriptPane = "Email Chat Transcript Pane";
-    public static readonly InputValidationPaneInput = "Email this chat transcript. This will be sent after your chat ends. Email address text area";
+    public static readonly InputValidationPaneInput = "Please provide e-mail address to send transcript. The transcript will be sent after the chat ends. Email address text area";
     public static readonly Send = "Send";
+    public static readonly Save = "Save";
```

Comment:
We shouldn't add this here since this is only used in tests? We can just hardcode it in the tests